src/pages/TranslationsPage/TranslationForm.jsx

```diff
@@ -0,0 +1,78 @@
+import React, {Component} from "react";
+import {Form, Button} from 'react-bootstrap'
+
+import {Header} from "../../components/Header";
+import {Footer} from "../../components/Footer";
+
+class TranslationForm extends Component{
+      constructor(props){
+        super(props)
+        this.state ={
+            track_id:'',
+            text:'',
+            language:'en',
+            auto_translate:"true"
+        }
+    }
+
+    changeHandler = (e) =>{
+        this.setState({[e.target.name]: e.target.value})
+    }
+
+    submitHandler = e => {
+        e.preventDefault()
+        this.state.track_id = this.props.location.state
+        this.state.auto_translate = (this.state.auto_translate==="true")
```

Comment:
hmm, this looks weird. maybe it's rather the job of the change handler to cast to the appropriate types what's coming from the controls to the state? @daniilzinevich can advise more